src/odbc/winsetup.c

```diff
@@ -318,7 +318,12 @@ ConfigDSN(HWND hwndParent, WORD fRequest, LPCSTR lpszDriver, LPCSTR lpszAttribut
 	/* Maybe allow the user to edit it */
 	if (hwndParent && fRequest != ODBC_REMOVE_DSN) {
 		INT_PTR result;
+#ifdef DLL_EXPORT
```

Comment:
I think all code in this file is exposing something for ODBC DM which cannot be used in case we are not a DLL.

Reply:
Further conditionalization may well be possible; I erred on the side of keeping more in.

Reply:
Merged the win* part.